src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -29,6 +30,10 @@ function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingE
     const [onboardingCompanySize] = useOnyx(ONYXKEYS.ONBOARDING_COMPANY_SIZE);
     const [onboardingPurposeSelected] = useOnyx(ONYXKEYS.ONBOARDING_PURPOSE_SELECTED);
     const [onboardingPolicyID] = useOnyx(ONYXKEYS.ONBOARDING_POLICY_ID);
+    const [allPolicies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
+
+    const paidGroupPolicy = Object.values(allPolicies ?? {}).find(PolicyUtils.isPaidGroupPolicy);
```

Comment:
we filter out to get the first paid policy. If the user is invited to a policy with a user role, the paid policy can be the policy that the user is not an admin then the onboarding [API is called failure.](https://github.com/Expensify/App/issues/54849#issuecomment-2602987070).
https://github.com/Expensify/App/issues/54849